backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -50,18 +50,31 @@ object BackendSettings extends BackendSettingsApi {
     // Validate if all types are supported.
     def validateTypes: Boolean = {
       // Collect unsupported types.
-      fields.map(_.dataType).collect {
-        case _: ArrayType =>
-        case mapType: MapType if mapType.keyType.isInstanceOf[StructType] =>
+      val unsupportedDataTypes = fields.map(_.dataType).collect {
+        case _: ByteType => "ByteType"
+        case _: ArrayType => "ArrayType"
+        case mapType: MapType if mapType.keyType.isInstanceOf[StructType] => "StructType as Key in MapType"
         // Parquet scan of nested map with struct as key type is not supported in Velox.
-        case _: ByteType =>
-      }.isEmpty
+      }
+      for (unsupportedDataType <- unsupportedDataTypes) {
+        // scalastyle:off println
+        println(
+          s"Validation failed for ${this.getClass.toString}" +
+            s"due to Not supported: data type $unsupportedDataType. in file schema. ")
```

Comment:
Should leave a space before "due to...", same to the below changes.

Reply:
@yma11 make sure all output have the same format, then we can easily extract info like fallback_reason = l.split(":")[-1]

Reply:
> Should leave a space before "due to...", same to the below changes. Please do an overall check.

Already noticed this and updated. Now still need do a rebase.

Reply:
> @yma11 make sure all output have the same format, then we can easily extract info like fallback_reason = l.split(":")[-1]

When we need to have a `split`? Each log is not long and I think we can just use the keyword `due to` to filter them out.